src/NServiceBus.Core/Transports/TransportReceiveInfrastructure.cs

```diff
@@ -10,6 +10,9 @@ public class TransportReceiveInfrastructure
     {
         /// <summary>
         /// Creates new result.
+        /// <param name="messagePumpFactory">Factory for creating the message pump.</param>
```

Comment:
Maybe clarify that this factory method is usually called multiple times? e.g.
```suggestion
        /// <param name="messagePumpFactory">Factory for creating a message pump for each queue to be consumed from.</param>
```

Reply:
👍 I'll the same to the public property.